packages/webapp/src/_api/handlers/ipfs-upload-handler.ts

```diff
@@ -1,23 +1,44 @@
 import { eosDefaultApi, eosJsonRpc } from "_app";
-import { edenContractAccount, ipfsConfig } from "config";
+import { edenContractAccount, ipfsApiBaseUrl, ipfsConfig } from "config";
 
 import { BadRequestError, InternalServerError } from "../error-handlers";
 import { IpfsPostRequest } from "../schemas";
 
+interface ActionIpfsData {
+    cid: string;
+    contract: string;
+    action: string;
+}
+
+interface ValidUploadActions {
+    [contract: string]: {
+        [action: string]: { maxSize: number };
+    };
+}
+
+const VALID_UPLOAD_ACTIONS: ValidUploadActions = {
+    [edenContractAccount]: {
+        inductprofil: { maxSize: 1_000_000 },
+        inductvideo: { maxSize: 100_000_000 },
```

Comment:
Should we move these into the config?

Reply:
...and reference them in the three places they're also used below?

Reply:
good idea! done!